### DISCUSSION TITLE
Vue 3 SSR

### DISCUSSION THREAD

**Discussion by noor-tg** [vuejs/rfcs#252] [General]
I did not find any documentation related to vue 3 server-side rendering . except for code in [server-side renderer](https://github.com/vuejs/vue-next/tree/master/packages/server-renderer) . and some thing about `createMemoryHistory` in [router-next documentation](https://next.router.vuejs.org/guide/migration/index.html). but nothing more.
any detailed guidelines to make ssr work with vue 3 ?

**Comment by hannoeru** [comment]
Here is some example in vite repo:
https://github.com/vitejs/vite/tree/main/packages/playground/ssr

**Reply by leomp12** [reply]
https://github.com/vitejs/vite/tree/main/packages/playground/ssr-vue
